traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -27,104 +29,103 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/lib/pq"
-
 	"github.com/apache/trafficcontrol/lib/go-log"
 	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/lib/go-tc/tovalidate"
 	"github.com/apache/trafficcontrol/lib/go-util"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/auth"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/dbhelpers"
-
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/deliveryservice"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/tenant"
-	"github.com/go-ozzo/ozzo-validation"
+
+	validation "github.com/go-ozzo/ozzo-validation"
 	"github.com/go-ozzo/ozzo-validation/is"
 	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
 )
 
-//we need a type alias to define functions on
+// A TOServer combines data about a server with metadata from an API request and
+// provides methods that implement several interfaces from the api package.
 type TOServer struct {
 	api.APIInfoImpl `json:"-"`
 	tc.ServerNullable
 }
 
-func (v *TOServer) SetLastUpdated(t tc.TimeNoMod) { v.LastUpdated = &t }
-func (v *TOServer) InsertQuery() string           { return insertQuery() }
-func (v *TOServer) UpdateQuery() string           { return updateQuery() }
-func (v *TOServer) DeleteQuery() string           { return deleteQuery() }
+func (s *TOServer) SetLastUpdated(t tc.TimeNoMod) { s.LastUpdated = &t }
+func (s *TOServer) InsertQuery() string           { return insertQuery() }
+func (s *TOServer) UpdateQuery() string           { return updateQuery() }
+func (s *TOServer) DeleteQuery() string           { return deleteQuery() }
```

Comment:
I actually don't think these last three need reference bindings at all, they could just be:
```go
func (*TOServer) InsertQuery() string           { return insertQuery() }
func (*TOServer) UpdateQuery() string           { return updateQuery() }
func (*TOServer) DeleteQuery() string           { return deleteQuery() }
```

Reply:
Good point.